r10k.gemspec

```diff
@@ -43,7 +43,7 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency 'yard', '~> 0.9.11'
 
-  s.files        = %x[git ls-files].split($/)
+  s.files        = %x[git ls-files].split($/).reject { |f| f.match(%r{^spec}) }
```

Comment:
Shelling out to git seems like a very strange way to determine what files to ship as the gem artifact. Would it not better to just explicitly ship the code we expect to be needed in the gem? maybe something like:
```ruby
s.files = Dir['bin/**'] + Dir['lib/**']
```
Using ^^ pattern would also avoid shipping the files under `docker` `docs` `integration` etc (which the filter in this pr does not exclude).

Reply:
@donoghuc Feel free to open a PR if you wanna change this. 😜